dojo/settings/settings.dist.py

```diff
@@ -1326,7 +1325,7 @@ def saml2_attrib_map_format(din):
     "HCLAppScan XML": ["title", "description"],
     "HCL AppScan on Cloud SAST XML": ["title", "file_path", "line", "severity"],
     "KICS Scan": ["file_path", "line", "severity", "description", "title"],
-    "MobSF Scan": ["title", "description", "severity"],
+    "MobSF Scan": ["title", "severity", "cwe", "file_path", "description"],
```

Comment:
I think this change means a note should be added to the upgrade notes to recalculate the hash codes?

Reply:
I will add this once it is clear when this will be released.

Reply:
@Maffooch fyi: When do you plan to release this? Then, I can add the info.

Reply:
2.47.0 looks quite busy, so I'll connect it to 2.47.1

Reply:
Target branch is dev, because of the database migration. So lets say 2.48.0, right?